### PR TITLE
Chpldoc modules/standard/Memory.chpl.

### PR DESCRIPTION
**Pull request by gbtitus**
Chpldoc modules/standard/Memory.chpl.

### CODE REVIEW

File: modules/standard/Memory.chpl [hunk 1]
```diff
@@ -17,10 +17,96 @@
...
+  regarding memory usage.  Except for :proc:`locale.physicalMemory` as
+  noted below, all of the procedures in :mod:`Memory` require memory
+  tracking to be enabled.  This can be done by setting at least one of
+  the following configuration variables at execution time, using either
+  the ``--configVarName=value`` or ``-sconfigVarName=value`` command
+  line options when a program is run.
+
+  ``memTrack``: :type:`bool`:
+    Enable memory tracking.  This causes memory allocations and
+    deallocations to be internally tracked, in order to support
```
Comment (hildeth) [inline modules/standard/Memory.chpl]: Nit: Remove "in order" for terseness.

File: modules/standard/Memory.chpl [hunk 2]
```diff
@@ -17,10 +17,96 @@
...
+/*
+  How much physical memory is present on this locale?
+
+  Note that this is a physical quantity and does not take into account
+  things like virtual memory support which might allow for allocating
+  a larger amount, or system or user limits which might prevent
+  allocating so much.
+
+  *Note:* This is the only procedure in the :mod:`Memory` module which
+  does not need memory tracking enabled in order to work properly.
```
Comment (hildeth) [inline modules/standard/Memory.chpl]: "to work properly" raises the question of what happens if memory tracking is not enabled.  I suggest rewording more vaguely, like: "... which does not rely on memory tracking being enabled."

**Comment by hildeth** [comment]
Looks fine to me, as-is.  My comments are advisory only.

**Comment by gbtitus** [comment]
Both line comments addressed -- thanks!
